tasks/Shopping Plan.md

```diff
@@ -0,0 +1,32 @@
+# Summary
+>Individuals were assigned a number of tasks to complete alone and with a group to determine if there is a level of intelligence among groups as there is in an individual.
+
+# References
+> Source: https://science.sciencemag.org/content/330/6004/686.full
+> Supporting: https://science.sciencemag.org/content/sci/suppl/2010/09/29/science.1193147.DC1/Woolley_SOM.pdf
+
+# Stimuli
+## The visual components
+
+## Materials for alternative versions of the experiment 
+
+
+# Procedure
+## Steps
+>The group had to plan a shopping trip as though they were all residents of the same house sharing the same car. Each member had a grocery list and the group was given a map that showed distances and times between each store.
+>The group had to decide how to get the items on their list while considering the constraints like some stores having cheaper and higher quality options requiring more driving time.
+>The point values were assigned to grocery items based on (a) each item purchased = +1 point, (b) bonus for high quality item = +1 point, (c) bonus for lower priced alternative = +2 points, (d) penalty for leaving frozen items in the car beyond 30 minutes = loss of all points for that item
+> The goal was to purchase as many high-quality items as possible, therefore aiming for the highest number of points.
+
+## Rolls 
+> Each individual had the same role as residents of the same house sharing the same car.
+
+## Instructions
+> No further instructions were provided other than what was listed in the procedure.
+
+# Criteria
+## Performance calculation
+> Performance was evaluated based on the number of points the group obtained.
+
```

Comment:
Should we include the measures of individual intelligence taken at the beginning of a session and the checkers taken after each activity?